Makefile

```diff
@@ -324,7 +336,14 @@ ptestoptionallong-nodoc:
 
 ###############################################################################
 
-configure: bootstrap src/doc/bootstrap configure.ac src/bin/sage-version.sh m4/*.m4 build/pkgs/*/spkg-configure.m4 build/pkgs/*/type build/pkgs/*/install-requires.txt build/pkgs/*/package-version.txt build/pkgs/*/distros/*.txt
+# The 'configure' script is just one of the files generated by 'bootstrap'.
+# CONFIGURE_DEPENDENCIES is the list of files that influence the generation of 'configure'.
+CONFIGURE_DEPENDENCIES=configure.ac src/bin/sage-version.sh m4/*.m4 build/pkgs/*/spkg-configure.m4 \
+	build/pkgs/*/type build/pkgs/*/SPKG.rst build/pkgs/*/checksums.ini build/pkgs/*/requirements.txt build/pkgs/*/install-requires.txt build/pkgs/*/package-version.txt build/pkgs/*/spkg-install build/pkgs/*/spkg-install.in
+# SPKG_INFO_DEPENDENCIES is the list of files that influence the run of 'sage-spkg-info' and hence
+# the generation of the files generated in 'src/doc' by 'src/doc/bootstrap'.
+SPKG_INFO_DEPENDENCIES=build/pkgs/*/type build/pkgs/*/SPKG.rst build/pkgs/*/requirements.txt build/pkgs/*/install-requires.txt build/pkgs/*/package-version.txt build/pkgs/*/distros/*.txt
+configure: bootstrap src/doc/bootstrap $(CONFIGURE_DEPENDENCIES) $(SPKG_INFO_DEPENDENCIES)
 	./bootstrap -d
 
```

Comment:
To wrap the long list:
```makefile
# The 'configure' script is just one of the files generated by 'bootstrap'.
# CONFIGURE_DEPENDENCIES is the list of files that influence the generation of 'configure'.
CONFIGURE_DEPENDENCIES=configure.ac src/bin/sage-version.sh m4/*.m4 \
	build/pkgs/*/spkg-configure.m4 build/pkgs/*/spkg-install build/pkgs/*/spkg-install.in build/pkgs/*/checksums.ini \
	build/pkgs/*/type build/pkgs/*/SPKG.rst build/pkgs/*/requirements.txt \
	build/pkgs/*/install-requires.txt build/pkgs/*/package-version.txt

# SPKG_INFO_DEPENDENCIES is the list of files that influence the run of 'sage-spkg-info' and hence
# the generation of the files generated in 'src/doc' by 'src/doc/bootstrap'.
SPKG_INFO_DEPENDENCIES=build/pkgs/*/distros/*.txt \
	build/pkgs/*/type build/pkgs/*/SPKG.rst build/pkgs/*/requirements.txt \
	build/pkgs/*/install-requires.txt build/pkgs/*/package-version.txt

configure: bootstrap src/doc/bootstrap $(CONFIGURE_DEPENDENCIES) $(SPKG_INFO_DEPENDENCIES)
	./bootstrap -d
```                       
and compare. Doesn't this ` build/pkgs/*/distros/*.txt` influence the generation of `configure`?

Reply:
> Doesn't this ` build/pkgs/*/distros/*.txt` influence the generation of `configure`?

No. `configure` only uses that at its runtime when it calls `sage-get-system-packages`

Reply:
Would you do the wrapping I suggested?

Otherwise it looks good to me.

Reply:
I've reformatted in a similar way in 663c222

Reply:
I wanted it to be easy to compare `CONFIGURE_DEPENDENCIES` and `SPKG_INFO_DEPENDENCIES`. But that is a minor thing.

Better than before. Thanks.